tangelo/linq/translator/translate_qubitop.py

```diff
@@ -50,10 +62,15 @@ def translate_operator(qubit_operator, source, target, n_qubits=None):
         if source not in TO_TANGELO:
             raise NotImplementedError(f"Qubit operator conversion from {source} to {target} is not supported.")
         qubit_operator = TO_TANGELO[source](qubit_operator)
+        # Clean the very small terms.
+        qubit_operator.compress()
```

Comment:
How would you feel if, as a user, you pushed data from an existing project into tangelo using a translate function and got something that did not have the same number of terms ?

This function translates operators from a format to another, I do not think we should take the liberty to "clean things up", here.

Reply:
It was previously in each individual function, but I understand the argument of keeping it consistent with the input. I will remove that line.